### ISSUE TITLE
Garbled text on GUI title with nightly build v1.0.3

### ISSUE THREAD

**Issue by RodenLuo** [taichi-dev/taichi#5135]
**Describe the bug**
Garbled text on GUI title 

**To Reproduce**
```py
pip install -i https://pypi.taichi.graphics/simple/ taichi-nightly
```
Run [fem-explicit.py](https://github.com/taichiCourse01/--Deformables/blob/master/fem-explicit.py)

**Log/Screenshots**

![image](https://user-images.githubusercontent.com/8460424/173183311-3a1063b2-e16b-4d1d-8563-8cd371f1513d.png)

**Additional comments**
This is after I meet #5051 with v1.0.2 and do the nightly build installation.

**Comment by PENGUINLIONG** [comment]
It might relate to this change. Thanks for the report! ;)

https://github.com/taichi-dev/taichi/pull/4925/files#diff-524274fa9e379a2cacb848a2b9133b31f497e3ba50a8b87607c9d3ebf2a317c2

**Comment by bobcao3** [comment]
Fixed by #5155
